config/initializers/lograge.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.configure do
   config.lograge.base_controller_class = 'ActionController::API'
-  config.lograge.formatter = Lograge::Formatters::Logstash.new
+  config.lograge.formatter = Lograge::Formatters::Json.new
```

Comment:
Handles action_controller process_action subscription events (https://guides.rubyonrails.org/active_support_instrumentation.html#process-action-action-controller).

Reply:
Lograge actually wipes out all default controller logging - including support for `log_tags` (a poorly named config option for pulling out attributes from an inflight request object).

Reply:
This means we can't actually add request_ids without monkey patching various bits of our logging deps.